src/VisualStudio/Core/Test.Next/Services/LanguageServiceTests.cs

```diff
@@ -93,17 +93,17 @@ private async Task<ImmutableArray<SymbolInformation>> GetVsSearchResultsAsync(Te
 
             using (var jsonRpc = JsonRpc.Attach(await client.RequestServiceAsync(WellKnownServiceHubService.LanguageServer)))
             {
-                var result = await jsonRpc.InvokeWithCancellationAsync<JObject>(
+                // TODO: use InvokeWithCancellationAsync(targetName, arguments, cancellationToken)
+                // (https://github.com/dotnet/roslyn/issues/45980)
+                var result = await jsonRpc.InvokeAsync<JObject>(
                     Methods.InitializeName,
-                    new object[] { new InitializeParams() },
-                    CancellationToken.None);
```

Comment:
📝 `InvokeAsync` just forwards to `InvokeWithCancellationAsync` using `CancellationToken.None`. Consider keeping the original code for all these calls.

Reply:
OK